packages/protocol/contracts/common/ProxyCloneFactory.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.5.13;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+import "./InitializableProxy.sol";
+
+contract ProxyCloneFactory is Ownable {
+  address public proxyAddress;
+  event ProxyCreated(InitializableProxy proxy);
+
+  function() external payable {}
```

Comment:
noting this adds yet another address that attestation fees have to flow through

Reply:
Attestation fees are paid in cUSD, I will remove this line.